src/layout/DwindleLayout.cpp

```diff
@@ -338,7 +345,42 @@ void CHyprDwindleLayout::onWindowCreatedTiling(CWindow* pWindow) {
 
     const auto PFORCESPLIT = &g_pConfigManager->getConfigValuePtr("dwindle:force_split")->intValue;
 
-    if (*PFORCESPLIT == 0) {
+    bool       horizontalOverride = false;
```

Comment:
`override`? 0 / 1 / 2 ? pointless vars?

Reply:
Changed to 2 bools that handle the vertical and horizontal override